CodeEditModules/Modules/AppPreferences/src/Sections/GeneralPreferences/GeneralPreferencesView.swift

```diff
@@ -85,8 +85,18 @@ private extension GeneralPreferencesView {
                     .tag(AppPreferences.FileExtensions.showOnly)
             }
             .frame(width: inputWidth)
+            if case .showOnly = prefs.preferences.general.fileExtensions {
+                let showedFileExtensionsBinding = Binding(get: {
+                    prefs.preferences.general.fileExtensionsShowed.extensions.joined(separator: ", ")
+                }, set: { newValue in
+                    let components = newValue.components(separatedBy: ", ")
```

Comment:
It would be better to use the following:
```swift
newValue.components(separatedBy: ",").trimmingCharacters(in: .whitespacesAndNewlines)
```

That way we also respect values entered without a space character.